message-index/messages/GHC-68686/example1/after/Main.hs

```diff
@@ -0,0 +1,7 @@
+{-# LANGUAGE TemplateHaskell, GADTs #-}
+
+module Main where
+
+main = undefined
```

Comment:
This addition distracted me from the point of the example the first time around.

What about having `main = putStrLn "Please look at the LANGUAGE pragmas!"` in both versions instead?